chart/docs/ha-setup.md

```diff
@@ -0,0 +1,37 @@
+# Prometheus High-Availability
```

Comment:
is this file linked from anywhere. We should also note that this is unnecessary if using the tobs cli and  give instructions for which flag to pass to the cli

Reply:
Linked it in `chart/README.md`